robotoff/insights/annotate.py

```diff
@@ -23,6 +23,18 @@
 from robotoff.products import get_image_id, get_product
 from robotoff.utils import get_logger
 
+"""
+This file allows to annotate products.
+
+To check whether the annotation already exists or not (and save it and send it to the off database), use the following commands:
+    from robotoff.insights.annotate import InsightAnnotatorFactor
+    annotator = InsightAnnotatorFactory.get(insight_type)
+    annotator.annotate(insight: ProductInsight, annotation: int, update: bool = True, data: Optional[Dict] = None, auth: Optional[OFFAuthentication] = None, automatic: bool = False)
+
+If you don't want to add it to the off database but only save it for example in case of a vote, you can use SAVED_ANNOTATION_VOTE_RESULT to warn the annotation has been taken into account. 
```

Comment:
```suggestion
If you don't want to add it to the Open Food Facts (off) database but only save it for example in case of a vote, you can use SAVED_ANNOTATION_VOTE_RESULT to warn the annotation has been taken into account. 
```